<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UniMain</class>
 <widget class="QMainWindow" name="UniMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UniMain</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>553</width>
      <height>450</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Welcome to University &lt;/p&gt;&lt;p&gt;Desktop App&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="addCourse">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>Add a course</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="viewCourse">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>View courses</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="addProgram">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>Add a program</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="viewProgram">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>View programs</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="addFaculty">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>Add faculty</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="viewFaculty">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string>View faculty</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionNew_user"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFIle"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_user">
   <property name="text">
    <string>New User</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
